core/src/test/scala/org/apache/spark/MapStatusesSerDeserBenchmark.scala

```diff
@@ -28,7 +26,7 @@ import org.apache.spark.storage.BlockManagerId
  * Benchmark for MapStatuses serialization & deserialization performance.
  * {{{
  *   To run this benchmark:
- *   1. without sbt: bin/spark-submit --class <this class> --jars <core test jar>
+ *   1. without sbt: bin/spark-submit --class <this class> <spark core test jar>
```

Comment:
use command
```
bin/spark-submit --class org.apache.spark.MapStatusesSerDeserBenchmark --jars spark-core_2.12-3.2.0-SNAPSHOT-tests.jar 
Error: Missing application resource.
```
should remove `--jars` because  `MapStatusesSerDeserBenchmark` run in local mode.